Classes/Utility.php

```diff
@@ -73,7 +73,8 @@ public function convertToSafeString($processedTitle, $spaceCharacter = '-', $str
 			$processedTitle = $this->csConvertor->conv_case('utf-8', $processedTitle, 'toLower');
 		}
 		$processedTitle = strip_tags($processedTitle);
-		$processedTitle = preg_replace('/[ \-+_]+/', $spaceCharacter, $processedTitle);
+		// the space in the next line is a non-breaking space
+		$processedTitle = preg_replace('/[ \s\-+_]+/u', $spaceCharacter, $processedTitle);
```

Comment:
I afraid this can be lost if somebody copy/pastes, etc. Can we do using `\x` sequences?